src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $cell-size * 7 + $gap-size * 6;
+  padding: 30px;
+  margin: 0 auto;
+
+  &--start-day {
+    @each $day, $margin in $day__names {
+      &--#{$day} :first-child {
+        margin-left: $margin * $cell-size + $margin * $gap-size;
+      }
+    }
+  }
+
+  &--month-length {
+    @for $num from 28 through 31 {
+      &--#{$num} :nth-child(n + #{$num + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
```

Comment:
There is no need to use `flex`, just set `line-height` and `align-items`

Reply:
Thanks for your comments)